chartmogul.go

```diff
@@ -152,6 +158,14 @@ type AnchorCursor struct {
 	StartAfter string `json:"start-after,omitempty"`
 }
 
+type MetaCursor struct {
+	NextKey    uint32 `json:"next_key,omitempty"`
```

Comment:
`subscriptions_events` id column is bigint which is signed 8 bytes (64 bits). Perhaps we should use uint64 here? Similiar with below `PrevKey`, `BeforeKey` etc.

It's small price to pay after all 😄